src/main.rs

```diff
@@ -46,6 +46,8 @@ struct CliArgs {
   #[clap(short, long, value_parser)]
   targets: Vec<String>,
   #[clap(long, value_parser)]
+  disable_browserslist: bool,
```

Comment:
Can we make this `--no-browserslist`? I think that naming convention for CLI flags is more common.

Reply:
I'll turn it into `--browserslist` to keep the feature opt-in; avoiding the breaking change of applying browserslist default targets.

For a future lightningcss 2.0 we could reconsider enabling browserslist by default.